src/sage/geometry/polyhedron/library.py

```diff
@@ -2201,7 +2201,7 @@ def truncated_six_hundred_cell(self, exact=False, backend=None):
         It is possible to use the backend ``'normaliz'`` to get an exact
         representation::
 
-            sage: polytopes.truncated_six_hundred_cell(exact=True,backend='normaliz') # not tested - long time ~16sec
+            sage: polytopes.truncated_six_hundred_cell(exact=True,backend='normaliz')   # not tested - long time (16s)
```

Comment:
```suggestion
            sage: polytopes.truncated_six_hundred_cell(exact=True,backend='normaliz')  # not tested, long time (16s)
```
best to use exactly 2 spaces to separate `#` from the code before it; unless it's aligned with something else, then it can be more